lib/client.js

```diff
@@ -182,6 +182,46 @@ Client.prototype._defineMethod = function(method, location) {
   };
 };
 
+Client.prototype._isSequenceRequired = function(method) {
+  try {
+    var tns = this.wsdl.definitions.xmlns.tns;
+    var childrenName = this.wsdl.definitions.schemas[tns].complexTypes.pullFileParams.children[0].name;
+    if(childrenName === 'sequence') {
+      return true;
+    }
+    return false;
+  } catch(err) {
+    return false;
+  }
+};
+
+Client.prototype._setSequenceArgs = function(argsScheme, args) {
+  var result = {};
+  if(typeof argsScheme !== 'object') {
+    return args;
+  }
+  for (var partIndex in argsScheme) {
+    if(typeof args[partIndex] === 'undefined') {
+      continue;
+    }
+    if(typeof argsScheme[partIndex] !== 'object') {
+      result[partIndex] = args[partIndex];
+    }
+    else {
```

Comment:
Please, pull ` else {` to the same line as the closing curly-brace of the `if {}`

-->
```
if {
//code
} else {
// code
}
```